docs/src/modules/java/pages/spring-client.adoc

```diff
@@ -0,0 +1,80 @@
+This guide walks you through the process of consuming these services using Spring.
+
+== Consuming REST endpoints
+
+Create a separate new Springboot project.
+
+For consuming the REST endpoints, we can use various REST client libraries provided via spring or others.
+For instance libraries like RestTemplate, Webflux etc.
+
+In this guide we are looking at steps using a Spring client using Webflux library.
+
+----
+ <dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-webflux</artifactId>
+ </dependency>
+----
+
+Steps:
+
+- Define a Webclient and Configure it to use Akka Serverless service's host and port.
+
+[source,java,indent=0]
+.src/main/java/com/example/client/spring/rest/config/WebClientConfig.java
+----
+include::example$java-valueentity-counter-spring-client/src/main/java/com/example/client/spring/rest/config/WebClientConfig.java[tag=webclientconfig]
+----
+
+- Using the above defined client call a relevant API in Akka Serverless service.
+For e.g. GET, POST etc. endpoints.
+
+[source,java,indent=0]
+.src/main/java/com/example/client/spring/rest/service/CounterServiceImpl.java
+----
+include::example$java-valueentity-counter-spring-client/src/main/java/com/example/client/spring/rest/service/CounterServiceImpl.java[tag=getCurrentCounterCall]
+----
+
+== Consuming GRPC Services
+
+Create a separate new Springboot project.
+
+For Consuming GRPC Services of Akka Serverless service we need to generate grpc client stubs.
+
+You could use protobuf maven plugins for stub creation.
+
+----
+<groupId>org.xolstice.maven.plugins</groupId>
+<artifactId>protobuf-maven-plugin</artifactId>
+----
+
+Following steps walks you through steps for creating
+
+- Get all the proto files from relevant akka serverless service and copy under `src/main/proto` folder.
+- Generate client stubs by running `mvn compile` or `mvn protobuf:compile`
+- Define a Message channel with host and port of akka serverless service
+
+[source,java,indent=0]
+.src/main/java/com/example/client/spring/rest/config/GrpcClientConfig.java
+----
+include::example$java-valueentity-counter-spring-client/src/main/java/com/example/client/spring/rest/config/GrpcClientConfig.java[tag=grpcclientconfig]
+----
+
+- Build client service via connecting message channel and then using relevant client stub call the desired service/method.
+
+[source,java,indent=0]
+.src/main/java/com/example/client/spring/rest/service/GrpcClientService.java
+----
+include::example$java-valueentity-counter-spring-client/src/main/java/com/example/client/spring/rest/service/GrpcClientService.java[tag=decreaseCounterCall]
+----
+
+== Sample
+
+Please refer to following sample which has support for both REST and GRPC consumers.
+
+`java-valueentity-counter-spring-client` module under samples link in the following repo :
+
+* https://github.com/lightbend/akkaserverless-java-sdk/tree/main/samples[Spring Client Java Value Entity Counter,window="new"]
```

Comment:
Ok, now I see this. 
This is good for the client, but we can also do the same for now and point to the value-entity as well. 

Maybe we can put it on top of the page and tell users that they can just checkout the samples or implement by hand. 

Btw, let's link to the project itself and not the samples project

Reply:
Yes , moved the repo links to top of the page and also created repo link for akka serverless value entity application.